src/media/js/addon/components/addonSubnav.js

```diff
@@ -8,9 +8,9 @@ export default class AddonSubnav extends React.Component {
   render() {
     return (
       <Subnav>
-        <ReverseLink to="addon-dashboard">My Firefox OS Add-ons</ReverseLink>
```

Comment:
It's directly underneath an item that says that:

<img width="492" alt="screen shot 2015-09-24 at 11 36 09 am" src="https://cloud.githubusercontent.com/assets/23885/10081080/7fdb45ec-62b0-11e5-8b92-45109ff1b76f.png">

There's something to be said about wordiness here, I think.